t/app/controller/dashboard.t

```diff
@@ -531,12 +531,12 @@ FixMyStreet::override_config {
             desc => 'Selecting no state does nothing',
             p1 => {
                     state   => 'fixed - user',
-                    conf_dt => DateTime->now(),
+                    conf_dt => DateTime->now()->subtract( minutes => 1 ),
```

Comment:
If they're now sorted by time, is there a need to have this as well? Guess it doesn't matter, so might as well leave, but surely the two can't be in the same microsecond? :)